tools/harness-thci/OpenThread_WpanCtl.py

```diff
@@ -88,22 +71,24 @@ def __init__(self, **kwargs):
             self.handle = None
             self.AutoDUTEnable = False
             self._is_net = False  # whether device is through ser2net
-            self.logStatus = {
-                'stop': 'stop',
-                'running': 'running',
-                'pauseReq': 'pauseReq',
-                'paused': 'paused',
-            }
+            self.logStatus = {'stop': 'stop', 'running': 'running', 'pauseReq': 'pauseReq', 'paused': 'paused'}
             self.logThreadStatus = self.logStatus['stop']
-            self.connectType = (
-                (kwargs.get('Param5')).strip().lower()
-                if kwargs.get('Param5') is not None
-                else 'usb'
+            self.connectType = (kwargs.get('Param5')).strip().lower() if kwargs.get('Param5') is not None else 'usb'
+            (self.prompt, self.wpan_cmd_prefix, self.wpan_interface) = (
+                kwargs.get('Param8').strip().split(',') if kwargs.get('Param8') else ['#', 'wpanctl', 'wpan0']
             )
+            self.wpan_cmd_prefix += ' '
+            self.precmd = (kwargs.get('Param9')).strip().split(',') if kwargs.get('Param9') else []
             if self.connectType == 'ip':
                 self.dutIpv4 = kwargs.get('TelnetIP')
                 self.dutPort = kwargs.get('TelnetPort')
                 self.port = self.dutIpv4 + ':' + self.dutPort
+                self.username = kwargs.get('Param6').strip() if kwargs.get('Param6') else None
+                self.password = kwargs.get('Param7').strip() if kwargs.get('Param7') else None
+                if not self.password:
+                    self.auth_password = False
```

Comment:
nit: `self.auth_password` seems redundant ?
Why not just use `if not self.password:` instead of `if not self.auth_password:`